src/components/Gallery/Gallery.js

```diff
@@ -0,0 +1,33 @@
+import { ImageComponent } from '../ImageComponent/ImageComponent';
+
+//TODO Temporary image callback function
+const getImageOnClickCallback = (id) => {
+    return () => {
+        console.log(id);
+    }
+}
+//END
+
+const GalleryCollection = (images) => {
+    let html = '';
+
+    for (var i = 0; i < images.length; i++) {
```

Comment:
Shorter and more clean syntax
![thebest](https://user-images.githubusercontent.com/43120132/147101146-31f5488d-a830-481c-9c61-46eb0a26ac36.PNG)


Also use the "--" instead of "-" in class names, to stick to the BEM methodology

Reply:
Should I copy this code if even if I don't fully understand it?

Reply:
Nope, but I highly rekommend to try and research to undestand this :P

Reply:
Ok, I'll do it in my spare time.